test/system/400-unprivileged-access.bats

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env bats   -*- bats -*-
+#
+# Tests #2730 - regular users are not able to read/write container storage
+#
+
+load helpers
+
+@test "podman container storage is not accessible by unprivileged users" {
+    skip_if_rootless "test meaningless without suid"
+
+    run_podman run --name c_uidmap --uidmap 0:10000:10000 $IMAGE true
+
+    # FIXME: this doesn't work: the -v fails with EPERM. What is the
```

Comment:
this issue was fixed with https://github.com/containers/libpod/pull/2716

Reply:
could we re-enable also the version with a volume?

Reply:
Would you mind actually testing it? Because it doesn't work for me, and I'm working with a podman built on your PR, which is post-2716, and it fails exactly as specified in my comment.

Reply:
ok sorry, I've broken it again with some other changes I did on my PR.

Pushed a new version right now

Reply:
Thank you! That works. New PR pushed to your branch, with `-v`